testsuite/probe/bcache1-devicegraph.xml

```diff
@@ -166,14 +166,14 @@
       <uuid>6990d6be-8fa8-4be7-8cf3-66ad1448864e</uuid>
       <mountpoint>swap</mountpoint>
       <mount-by>uuid</mount-by>
-      <fstab-options>defaults</fstab-options>
+      <mount-opts>defaults</mount-opts>
```

Comment:
Would it not be more consistent to remove the "defaults" since it is also not present in the internal data structures?